__tests__/app.test.js

```diff
@@ -62,6 +62,14 @@ describe("GET /api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/15")
       .expect(400)
+      .then(({ body }) => {
```

Comment:
These need switching around 404 - article not found. 
You've asked for no 15 ( a valid number)

15 was not found 404